monai/utils/misc.py

```diff
@@ -61,3 +61,20 @@ def is_scalar(val):
     if torch.is_tensor(val) and val.ndim == 0:
         return True
     return np.isscalar(val)
+
+
+def process_bar(index, count, bar_len=30, newline=False):
+    """print a process bar to track some time consuming task.
+
+    Args:
+        index (int): current satus in process.
+        count (int): total steps of the process.
+        bar_len(int): the total length of the bar on screen, default is 30 char.
+        newline (bool): whether to print in a new line for every index.
+    """
+    end = '\r' if newline is False else '\n\r'
```

Comment:
If we print newline should this be "\r\n"? I think "\n\r" might have unintended effects.

Reply:
Hi @ericspod ,

Thanks for your comments, I tested both, didn't see any difference.
Do you think we still need to change to '\r\n'?
Thanks.

Reply:
It is the correct way, "\n" is the newline for the Unix world and "\r\n" is for Windows. In Unix the "\r" in "\n\r" is unnecessary.

Reply:
Hi @ericspod ,

OK, I already updated to '\r\n', could you please help review again?
Thanks.